Chat App/Chat App/ChatTableView/MessageTableViewCell.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  MessageTableViewCell.swift
+//  Chat App
+//
+//  Created by tornike on 04.05.23.
+//
+import UIKit
+
+final class MessageTableViewCell: UITableViewCell {
+    
+    // MARK: Components
+    private let messageLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.textColor = .black
+        return label
+    }()
+    
+    private let dateLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .gray
+        label.font = Constants.FontSize.dateLabelFontSize
+        return label
+    }()
+    
+    private let chatBubble: UIView = {
+        let bubble = UIView()
+        bubble.layer.cornerRadius = 25
+        bubble.layer.masksToBounds = true
+        bubble.backgroundColor = Constants.ChatBubbleColors.lightModeBubbleColor
+        return bubble
+    }()
+    
+    private let middleBubble: UIView = {
+        let bubble = UIView()
+        bubble.layer.cornerRadius = 12
+        bubble.layer.masksToBounds = true
+        bubble.backgroundColor = Constants.ChatBubbleColors.lightModeBubbleColor
+        return bubble
+    }()
+    
+    private let smallestBubble: UIView = {
+        let bubble = UIView()
+        bubble.layer.cornerRadius = 10
+        bubble.layer.masksToBounds = true
+        bubble.backgroundColor = Constants.ChatBubbleColors.lightModeBubbleColor
+        return bubble
+    }()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setUp()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: Setup
+    private func setUp() {
+        backgroundColor = .clear
+        setUpChatBubbles()
+        setUpMessageLabel()
+        setUpDateLabel()
+        setUpLayoutConstraints()
+    }
+    
+    private func setUpChatBubbles() {
+        contentView.addSubview(chatBubble)
+        contentView.addSubview(middleBubble)
+        contentView.addSubview(smallestBubble)
+        chatBubble.translatesAutoresizingMaskIntoConstraints = false
+        middleBubble.translatesAutoresizingMaskIntoConstraints = false
+        smallestBubble.translatesAutoresizingMaskIntoConstraints = false
+    }
```

Comment:
This way will be better I think: 

`[chatBubble, middleBubble, smallestBubble].forEach({`
`contentView.addSubivew($0)`
`$0.translatesAutoresizingMaskIntoConstraints = false`
`})`